classes/publication/Collector.php

```diff
@@ -79,6 +81,17 @@ public function filterBySubmissionIds(?array $submissionIds): self
         return $this;
     }
 
+    /**
+     * Set the filter for issue IDs to limit the publications retrieved.
+     *
+     * @param array|null $issueIds An array of issue IDs to filter by, or null to remove the filter.
```

Comment:
When possible, I think the shorter syntax is preferred:

```suggestion
     * @param ?int[] $issueIds An array of issue IDs to filter by, or null to remove the filter.
```